src/main/java/com/google/api/codegen/discovery/transformer/ruby/RubySampleMethodToViewTransformer.java

```diff
@@ -0,0 +1,180 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.discovery.transformer.ruby;
+
+import com.google.api.codegen.discovery.config.FieldInfo;
+import com.google.api.codegen.discovery.config.MethodInfo;
+import com.google.api.codegen.discovery.config.SampleConfig;
+import com.google.api.codegen.discovery.transformer.SampleMethodToViewTransformer;
+import com.google.api.codegen.discovery.transformer.SampleNamer;
+import com.google.api.codegen.discovery.transformer.SampleTransformerContext;
+import com.google.api.codegen.discovery.transformer.SampleTypeTable;
+import com.google.api.codegen.discovery.viewmodel.SampleAuthView;
+import com.google.api.codegen.discovery.viewmodel.SampleFieldView;
+import com.google.api.codegen.discovery.viewmodel.SamplePageStreamingView;
+import com.google.api.codegen.discovery.viewmodel.SampleView;
+import com.google.api.codegen.util.Name;
+import com.google.api.codegen.util.SymbolTable;
+import com.google.api.codegen.util.ruby.RubyTypeTable;
+import com.google.api.codegen.viewmodel.ViewModel;
+import com.google.common.collect.ImmutableMap;
+import com.google.protobuf.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+public class RubySampleMethodToViewTransformer implements SampleMethodToViewTransformer {
```

Comment:
A lot of the boilerplate in this function appears to be language independent. One thing to consider for the future is whether we could factor out that code (into, say, the parent class) and call it with the language-specific parameters.

Something to keep in mind for after this PR.